ext/pcntl/tests/pcntl_waitid.phpt

```diff
@@ -0,0 +1,30 @@
+--TEST--
+pcntl_waitid()
+--EXTENSIONS--
+pcntl
+posix
+--FILE--
+<?php
+$pid = pcntl_fork();
+if ($pid == -1) {
+    die("failed");
+} else if ($pid) {
+    $status = 0;
```

Comment:
once you re done we need to test pcntl_waitid also with incongruous values on purpose.